assets/src/js/menu_editor.js

```diff
@@ -347,5 +347,24 @@ const menuEditorRestrictions = () => {
 };
 
 document.addEventListener('DOMContentLoaded', () => {
-  menuEditorRestrictions();
+  // menuEditorRestrictions();
```

Comment:
This is code won't be included, I've pushed just for testing (commenting the `menuEditorRestrictions` function) since I'm leading with a strange behavior when I'm trying to show this message into the `nav-menu-footer`. 

For some reason the message disappeared (see the video). In the beginning I thought it was something related to the `MutationObserver` with something that is stored to the cache, but not, because I've tried using the Firefox, Safari, Chrome (private and incognito) and got the same behavior.


https://user-images.githubusercontent.com/77975803/174859285-69d93796-aaa8-4c35-8b98-45b96bb609c3.mov

Reply:
It happens to `div.notice` elements ([more info here](https://github.com/WordPress/wordpress-develop/blob/19e61fb14fab5a3fdf7af2aa27b8fce6d7f5a05e/src/js/_enqueues/admin/common.js#L1083)). 
It's solved by just adding `inline` or `below-h2` class names.